composer.json

```diff
@@ -37,7 +37,7 @@
         }
     },
     "require":           {
-        "php":                               "^5.6 || ^7.0",
+        "php":                               "^7.0",
```

Comment:
Active support for PHP 7.0 drops in 15 days, so I'd suggest to require 7.1 by default.